include/cvc5/cvc5_proof_rule.h

```diff
@@ -2310,7 +2310,14 @@ enum ENUM(ProofRewriteRule)
    * **Builtin -- Distinct elimination**
    *
    * .. math::
-   *   \texttt{distinct}(t_1, \ldots, tn) = \bigwedge_{i \neq j} t_i \neq t_j
+   *   \texttt{distinct}(t_1, t_2) = \neg (t_1 = t2)
+   *
+   * if :math:`n = 2`, or
+   * 
+   * .. math::
+   *   \texttt{distinct}(t_1, \ldots, tn) = \bigwedge_{i=1}^n \bigwedge_{j=i+1}^n t_i \neq t_j
+   *
+   * if :math:`n > 2`
```

Comment:
Does `n > 2` have to be a special case?

Reply:
Not really, although I usually prefer having two cases in a situation like this. Mainly because it is easier to read and there cannot be any confusion with the semantics of the corner cases of the big and.